src/menu_input.c

```diff
@@ -595,3 +583,587 @@ void AddMenuCursorSprite(struct MenuInputStruct *param_1)
 {
     AddMenuCursorSprite_(param_1, 0);
 }
+
+void AddMenuCursorSprite_(struct MenuInputStruct *a0, u32 a1)
+{
+    struct SpriteOAM sp = {};
+    struct SpriteOAM* ptr;
+    s32 value;
+    s32 r0;
+    s32 r1;
+    s32 r2;
+    s32 r5;
+
+    if (a0->unk1A > 0) {
+        UpdateMenuCursorSpriteCoords(a0);
+
+        if (!(a0->unk24 & 8)) {
+            register u32 tmp asm("r0"), tmp2 asm("r1");
```

Comment:
There are a couple `register` uses in this file that are missing `NONMATCHING`.

Reply:
I thought we agreed from a previous PR that register wouldn't need NONMATCHING since it was implied and would be searched for/removed later

Reply:
Did we agree on that? In the last PR, the discussion ended with adding the NONMATCHINGs. https://github.com/pret/pmd-red/pull/149#discussion_r1289507036